File: official/guides/customs-guide/java/create-customs-item.java
package customs;

import java.util.HashMap;

import com.easypost.EasyPost;
import com.easypost.exception.EasyPostException;

public class CreateCustomsItem {
    public static void main(String[] args) throws EasyPostException {
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");

        Map<String, Object> customsItemMap = new HashMap<String, Object>();
        customsItemMap.put("description", "T-shirt");
        customsItemMap.put("quantity", 1);
        customsItemMap.put("value", 11);
        customsItemMap.put("weight", 6);
        customsItemMap.put("origin_country", "US");
        customsItemMap.put("hs_tariff_number", "610910");

        CustomsItem customsItem1 = CustomsItem.create(customsItemMap);
    }
}

File: official/guides/customs-guide/csharp/create-customs-item.cs
CustomsItem customsItem1 = await CustomsItem.Create(new Dictionary<string, object>() {
  { "description", "T-shirt" },
  { "quantity", 1 },
  { "weight", 6 },
  { "value", 11 },
  { "origin_country", "US" },
  { "hs_tariff_number", "610910" }
});
